app/assets/config/manifest.js

```diff
@@ -0,0 +1,2 @@
+//= link graphiql/rails/application.css
+//= link graphiql/rails/application.js
```

Comment:
We followed a tutorial that instructed to add this because we were getting an error when trying to hit our query in graphiql.